test/fixture/assert.snap

```diff
@@ -0,0 +1,19 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
@novemberborn Can you open an issue on Jest to make it configurable? They seem open to making it more generic when needed.

I'm happy to land this as is for now, but long-term we'll want our own naming and versioning.